lib/vanagon/project.rb

```diff
@@ -476,5 +476,33 @@ def generate_package
     def generate_packaging_artifacts(workdir)
       @platform.generate_packaging_artifacts(workdir, @name, binding)
     end
+
+    # Generate a json file which lists all of the dependant components of the project
+    #
+    # @return [Hash] where the top level keys are components and their values
+    #   are hashes with additional information on the component.
+    def generate_dependencies_info
+      all_components = {}
+      @components.each do |component|
+        all_components = all_components.merge(component.get_dependency_hash)
+      end
+      all_components
+    end
+
+    # Writes a json file at `ext/build_metadata.json` containing information
+    # about what went into a built artifact
+    def generate_manifest_json
+      manifest = {
+        "packaging_type" => {
+          "vanagon" => Gem.loaded_specs["vanagon"].version.to_s,
+        },
+        "git_sha" => version,
+        "components" => generate_dependencies_info,
+        "build_time" => BUILD_TIME.iso8601,
```

Comment:
I see.

I'm not worried about the formats themselves, just the fact that we're using different ones to represent the same thing.

If we knew that it was better to use ISO over RFC then we should've used ISO in the ezbake change.

How are we going to be using these timestamps?  
Aren't we going to be comparing them?  
Will we ever be comparing the timestamp from ezbake with the timestamp from vanagon (i.e. we're comparing a timestamp in ISO format with a timestamp in RFC format)? 
Is that going to work properly, or will we need to convert one to the other?

Not trying to ask leading questions here; maybe all this will work seamlessly 😄